Monika After Story/game/script-stories.rpy

```diff
@@ -356,3 +356,42 @@ label mas_story_immortal_love:
     m 1eua "At least the ending was very heartwarming."
     m 1hua "Thanks for listening~"
     return
+
+init 5 python:
+    addEvent(Event(persistent._mas_story_database,eventlabel="mas_story_self_hate",
+        prompt="Self-hate",unlocked=False),eventdb=store.mas_stories.story_database)
+
+label mas_story_self_hate:
+
+    m 1esa "Once, there were two lovers who lived together."
+    m 1eub "They had been living together for a very long time and things were going well."
+    m 1hua "Life was simple, and they both loved eachother."
+    m 2eud "But one day, the man suddenly killed his lover for seemingly no reason."
+    m 3rksdlc "He {i}did{/i} love her, and in a healthy way too!"
+    m 1rksdlc "There was never any fight or argument either."
+    m 1ekc "There was nothing to gain from murdering her at all."
+    m 4rksdlc "No satisfaction, nothing out of the ordinary on her will..."
+    m 1dsc "And the man was left emotionally scarred, having lost the love of his life."
+    m "..."
+    m 1ekd "So why?"
+    m 1rksdlc "How could this happen all of a sudden?"
+    m 3rksdlc "It wasn't that he didn't love her."
+    m 3rksdla "It was because he loved her so, so very much."
+    m 2eka "He didn't hate her..."
+    m 2ekc "He just hated himself so much as to deny his own happiness."
+    m 2dsc "..."
+    if persistent._mas_pm_love_yourself == False:

```

Comment:
at a glance, I would immediately change this to `not persistent._mas_pm_love_yourself` since that is functionally equivalent. However, since this variable can be None, I see why you explicity checked for False. 

But to make this clearer, do `persistent._mas_pm_love_yourself is False`. This makes it obvious that what matters is that the value of this variable is False, not that it's truthiness is False.